apps/condo/domains/meter/schema/_internalDeleteMeterAndMeterReadingsService.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Generated by `createservice meter._internalDeleteMeterAndMeterReadingsService --type mutations`
+ */
+
+const { isEmpty } = require('lodash')
+
+const { getLogger } = require('@open-condo/keystone/logging')
+const { GQLCustomSchema, find } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/meter/access/_internalDeleteMeterAndMeterReadingsService')
+const { Meter } = require('@condo/domains/meter/utils/serverSchema')
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+
+const _internalDeleteMeterAndMeterReadingsService = new GQLCustomSchema('_internalDeleteMeterAndMeterReadingsService', {
+    schemaDoc: 'Mutation to delete meters and meter readings for property',
+    types: [
+        {
+            access: true,
+            type: 'input _internalDeleteMeterAndMeterReadingsInput { dv: Int!, sender: SenderFieldInput!, propertyIds: [String]! }',
+        },
+        {
+            access: true,
+            type: 'type _internalDeleteMeterAndMeterReadingsOutput { status: String! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.can_internalDeleteMeterAndMeterReadings,
+            schema: '_internalDeleteMeterAndMeterReadings(data: _internalDeleteMeterAndMeterReadingsInput!): _internalDeleteMeterAndMeterReadingsOutput',
+            resolver: async (parent, args, context) => {
+                const { data } = args
+                const { dv, sender, propertyIds } = data
+                const logger = getLogger('DeleteMeters')
```

Comment:
Why it has different name here? It's not obvious to find messages by `DeleteMeters` keyword instead of `_internalDeleteMeterAndMeterReadings`. It may be fine if I can find mutation name directly in it.

Reply:
And why logger is created on every single call?